### DISCUSSION TITLE
Can I Get co-pilot for free?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#143202] [Copilot Conversations]
### Select Topic Area

Question

### Body

What i have to do to get free co-pilot access?

**Comment by uttamsdev** [comment]
Github co-pilot is a premium tool you have to buy it usually. But if you are a student with having an university email, you can simply apply for gihtub student developer pack. You  will get co-pilot there for free.

**Reply by ghost** [reply]
Thanks.

**Comment by uttamsdev** [comment]
Hope you understood.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/143204#discussioncomment-11116708) for reference to why this discussion was closed and locked. Thanks!
